docs/how-to/spacetimedb-standalone.md

```diff
@@ -0,0 +1,250 @@
+# Self Hosting SpacetimeDB
+
+This tutorial will guide you through setting up SpacetimeDB on an Ubuntu 24.04 server, securing it with HTTPS using Nginx and Let's Encrypt, and configuring a systemd service to keep it running.
+
+## Prerequisites
+- A fresh Ubuntu 24.04 server (VM or cloud instance of your choice)
+- A domain name (e.g., `example.com`)
+- `sudo` privileges on the server
+
+Also, join the community! We have an official discord server: https://discord.gg/spacetimedb
```

Comment:
This feels weird and breaks up the flow of the tutorial. We tell people to join the Discord plenty; we don't need to do it here too.

Reply:
I think the CTA is important here - I think more people will use this guide as it's been frequently requested. This install is also slightly more complicated and I want people to join the discord if they run into issues. We could rephrase to something like, "If you run into issues, reach out to us in Discord! <link>" What do you think?